planner/core/src/main/java/eu/seaclouds/platform/planner/core/Planner.java

```diff
@@ -105,6 +108,7 @@ private String generatMMOutput(Map<String, HashSet<String>> mmResult, Map<String
         return mapper.writerWithDefaultPrettyPrinter().writeValueAsString(singleRes);
     }
 
+
     public String plan() throws ParsingException, IOException {
```

Comment:
That method generate a json String so String seems good to me.
The parameters are not needed as they are in the state of the planner.

How that improves testability?